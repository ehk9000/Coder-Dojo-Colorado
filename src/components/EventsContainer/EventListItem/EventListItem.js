import React from 'react'

function EventListItem() {
  return (
    <div>
      
    </div>
  )
}

export default EventListItem
