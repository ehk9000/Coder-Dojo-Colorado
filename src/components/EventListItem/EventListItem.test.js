import React from 'react';
import { shallow } from 'enzyme';
import EventListItem from './EventListItem';

describe('EventListItem', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<EventListItem />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
