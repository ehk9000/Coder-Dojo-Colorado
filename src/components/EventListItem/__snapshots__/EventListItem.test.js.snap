// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventListItem should match snapshot 1`] = `ShallowWrapper {}`;
